app/src/main/java/com/example/lwilcox/lab2/ImageLoadTask.java

```diff
@@ -0,0 +1,44 @@
+package com.example.lwilcox.lab2;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.os.AsyncTask;
+import android.widget.ImageView;
+import java.io.InputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+/**
+ * Created by lwilcox on 9/23/2015.
+ */
+public class ImageLoadTask extends AsyncTask<Void, Void, Bitmap> {
+    private String url;
+    private ImageView imageView;
+    public ImageLoadTask(String url, ImageView imageView) {
+        this.url = url;
+        this.imageView = imageView;
+    }
+
+    @Override
+    protected Bitmap doInBackground(Void... params) {
+        try {
+            URL urlConnection = new URL(url);
+            HttpURLConnection connection = (HttpURLConnection) urlConnection
+                    .openConnection();
+            connection.setDoInput(true);
+            connection.connect();
+            InputStream input = connection.getInputStream();
+            Bitmap myBitmap = BitmapFactory.decodeStream(input);
+            return myBitmap;
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+        return null;
```

Comment:
Although functionally the same it is clearer if you move this return null into the catch block.